### ISSUE TITLE
Request: merge master with wx3 branch more often

### ISSUE THREAD

**Issue by poettler-ric** [DarkFenX/Pyfa#338]
Problem
=======
Currently I am running Fedora 21 (wich will/might hit it's end-of-life in November 2015). The only thing blocking me from upgrading my system is pyfa since Fedora 22+ only ships with wxPython 3.

Tried solutions
===============
First I tried to use the vanilla wx3 branch but it seems to be based on an older version of pyfa.

Then I tried to merge master into wx3. After one hour of trying and not knowing what I was doing (since I got no idea why each file was changed in the respective branch) I gave up.

I am also not sure whether the wx3 branch only contains ui changes concerning wxPython 3 since there is a ``setup.py`` file in that branch which is usually used to install the program. 

Request
=======
It would be great if once a release is tagged the tag is merged into branch wx3.
I know I am asking since it is a lot of work but it might enable more people using Linux to test the code and maybe help to determine the cause of some problems.

**Comment by blitzmann** [comment]
1) You could always compile and run wxPython 2.8 on the new Fedora system (but no, this isn't optimal)

2) setup.py is Kadesh's thing. He was attempting a new way to build packages. It should not be considered stable or useable

3) All said and done, I agree that the wx3 branch should be updated more often. This is simply a matter of us remembering to do it, which I cannot promise. But I will try to at least merge releases with wx3 from now on.

The wx3 branch for Linux is currently in a pretty good shape - the only known issue is preferences (#308). There's a few other problems on OS X if you have that and want to look into it - this is the hardest platform to test on for us as I'm the only one that has access to OS X, and that's on a slow VM. 

https://github.com/DarkFenX/Pyfa/labels/wx3

**Comment by nizmogtr** [comment]
I have an os x 10.10 machine and am willing to help out. I have the current wx3 build working right now. What do you need me to do to help out?

**Comment by blitzmann** [comment]
The OS X-specific issues are #305 and #306

**Comment by poettler-ric** [comment]
Thanks for the merge. Currently I am pretty busy but today I've set up one of my machines to F22 for easier testing. Once I have packaged it up I will try to investigate this preferences thing.
